guides/user/canary/stage/index.md

```diff
@@ -0,0 +1,169 @@
+other permutations, like multiple stages? -->
+
+1. In the pipeline in which you will run the canary, click __Add stage__.
+
+   This pipeline needs to be in an application that has access to the [canary
+   configuration](/guides/user/canary/config/) you want to use.
+
+1. For __Type__ select __Canary__.
+
+1. Give the stage a name, and use the __Depends On__ field to position the stage
+downstream of its dependencies.
+
+   ![Canary stage declaration](/guides/user/canary/stage/canary_stage_top.png)
+
+1. Select the __Analysis Type__&mdash;either __Real Time__ or __Retrospective__.
+
+   * __Real Time__
+
+     The analysis happens for a specified time period, beginning when the stage
+     executes.
+
+     For __Real Time__, also specify the number of hours to run (__Lifetime__).
+
+   * __Retrospective__
+
+     Analysis occurs over some specified period. Typically, this is done
+     for a time period in the past, against a baseline deployment and a canary
+     deployment which have already been running before this canary analysis
+     stage starts.
+
+1. Specify the analysis configuration:
+
+   * Choose the __Config Name__.
+
+     This is the canary config you created [here](). That configuration must be
+     created within this same application, unless your Spinnaker is set up so
+     that all canary configurations are available to all pipelines.
+
+   * If you're running a __Real Time__ analysis, set a __Delay__.
+
+     This is the number of minutes to wait before starting the analysis. This
+     gives both the baseline and canary a chance to get warmed up before they're
+     expected to provide meaningful  metrics. Leave blank for a delay of zero
+     minutes.
+
+   * Set a __Delay__.
+
+     For real-time analyses, how many minutes to wait before starting the
+     analysis. The gives both the baseline and canary a chance to get "warmed
+     up" before they're expected to provide meaningful metrics.
+
+   * Set the __Interval__.
+
+     This is how frequently (in minutes) to capture and score the metrics.
+
+   * For __Analysis Type__, select __Growing__ or __Sliding Lookback__.
+
+     - In a growing analysis, a judgment is taken every [interval] minutes, but
+     each judgment goes all the way back to the beginning of the __Lifetime__.
+
+     - A sliding Lookback also makes a judgment every [interval], but each
+     judgment only looks at the data from the most recent lookback duration.
+     It would not be unusual for the __Interval__ and the __look-back__
+     duration to be the same, but they don't have to be.
+
+   ![Canary stage declaration](/guides/user/canary/stage/stage_config_analysis.png)
+
+1. Describe the metric scope.
+
+   You can enter server groups and regions by name, but you can also click the
+   magic wand here to automatically populate them with pipeline expressions that
+   resolve to those names, from an upstream Clone Server Group stage.
```

Comment:
Treat these expressions as a starting point from which you can then customize them for your particular use case.